fixed-hash/Cargo.toml

```diff
@@ -18,7 +18,7 @@ rand = { version = "0.7", optional = true, default-features = false }
 rustc-hex = { version = "2.0", optional = true, default-features = false }
 quickcheck = { version = "0.9", optional = true }
 byteorder = { version = "1.2", optional = true, default-features = false }
-static_assertions = "0.3"
+static_assertions = "1.0.0"
```

Comment:
I'm afraid this is technically a breaking change, since we `pub use static_assertions;`, but this is unlikely to be a problem in practice.